sdk/src/user.ts

```diff
@@ -1196,116 +1196,157 @@ export class User {
 		return health;
 	}
 
-	/**
-	 * calculates sum of position value across all positions in margin system
-	 * @returns : Precision QUOTE_PRECISION
-	 */
-	getTotalPerpPositionValue(
+	calculatePerpPositionLiabilityWeight(
```

Comment:
i would rename this to 
`calculateWeightedPerpPositionValue` ?

Reply:
+1